### PR TITLE
fix display for network change requests

### PR DESCRIPTION
**Pull request by skylarbarrera**
## What changed (plus any additional context for devs)
we we're showing the wc request network but in this case we want to show the network they are switching to 

casting chainId to number following: https://github.com/rainbow-me/rainbow/pull/2688

## PoW (screenshots / screen recordings)

https://cloud.skylarbarrera.com/Screen-Shot-2021-12-16-12-13-07.01.png

## Final checklist
[x] Assigned individual reviewers?
[x] Added labels?

### CODE REVIEW

File: src/screens/WalletConnectApprovalSheet.js [hunk 1]
```diff
@@ -425,14 +425,22 @@ export default function WalletConnectApprovalSheet() {
                 <ButtonPressAnimation>
                   <Row marginTop={android ? -10 : 0}>
                     <Centered marginRight={5}>
-                      <ChainLogo network={approvalNetworkInfo.value} />
+                      <ChainLogo
+                        network={
+                          type === WalletConnectApprovalSheetType.connect
```
Comment (skylarbarrera) [inline src/screens/WalletConnectApprovalSheet.js:430]: this follows the same pattern we use on line 337

File: src/utils/ethereumUtils.ts [hunk 2]
```diff
@@ -288,7 +289,8 @@ const getNetworkFromChainId = (chainId: number): Network => {
  */
 const getNetworkNameFromChainId = (chainId: number): string | undefined => {
```
Comment (skylarbarrera) [inline src/utils/ethereumUtils.ts:290]: we don't use this util in anywhere else so im tweaking to use the display name(NetworkInfo) vs the underlying network name(chains.json)

**Review by brunobar79 — APPROVED** [review]
👍

**Review by benisgold — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]
lgtm

**Pull request merged**